models/sketch.test.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const Sketch = require('./sketch');
+
+describe('sketch model', () => {
+    it('should invalidate if length is undefined', () => {
+        const s = new Sketch({
+            octave: 1,
+            pitch: 'a'
+        });
+
+        s.validate(error => expect(error.errors.length).toBeTruthy());
+    });
+
+    it('should invalidate if length is not a number', () => {
+        const s = new Sketch({
+            length: {},
+            octave: 1,
+            pitch: 'a'
```

Comment:
may want to get more granular and expect certain specific strings e.g. `('a', 'b', 'c', ...'g');`